cargo-cyclonedx/src/cli.rs

```diff
@@ -37,6 +37,20 @@ pub struct Args {
     #[clap(long = "quiet", short = 'q')]
     pub quiet: bool,
 
+    // The feature selection flags are not mutually exclusive in Cargo,
+    // so we keep the same behavior here too.
```

Comment:
I only understand this comment after having read all the code and did some research myself.

This code does not belong to `all_features`, it means that all three of these new flags are not mutually exclusive.
If that understanding is correct: I think it's a useful comment to have but it needs clarification.

Example: "The three feature selection flags (all_features, no_default_features, features) are not ..."